schema/source_schema.json

```diff
@@ -185,6 +185,19 @@
         "desc": {
           "type": "string",
           "description": "Human readable field name"
+        },
+        "regex": {
+          "type": "string",
+          "description": "Regular expression that may match the field contents.",
+          "format": "regex"
+        },
+        "alias": {
+          "type": "array",
+          "items": {
+            "description": "Regular expressions that may match the field.",
+            "type": "string",
+            "format": "regex"
+          }
         }
```

Comment:
Should we have the new `values` property for the id `field`?
```suggestion
          }
        },
        "values": {
          "type": "array",
          "items": {
            "description": "Actual identifiers for the boundaries of the dataset",
            "type": "string"
          }
        }
```

Reply:
Interesting idea. I hadn't considered adding `values` to the source files. I just assumed we'd only populate the `values` in the destination manifest from the geojson files.

Do you see a use case for adding a `values` array to the source files? I suppose it could be used to strictly validate the GeoJSON files, but that may be out of scope for this PR.

Reply:
Apologies Nick, I totally confused the schema of the source and the manifest :facepalm: 

![](https://media.giphy.com/media/ieEGqor2h3dIoJdNbS/giphy.gif)